content/docs/reference/spec/migration/buildpack-api-0.4-0.5.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide is most relevant to stack and buildpack authors. 
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) for buildpack API 0.5 for the full list of changes and further details.
+
+## Stack author
+
+### Stack ID
+Stack ID requirements have been removed from the buildpack specification.
+Each stack must indicate either a compatible stack and provide a valid stack ID, or compatibility with any stack by specifying the special value `*`.
```

Comment:
```suggestion
The buildpack may now indicate compatibility with any stack by specifying the special value `*`.
```

1. First sentence made me think that something is going away when in reality it's simply loosening a requirement.
2. It's really a buildpack that is essentially stating that it's _compatible_ with stacks. The only change is that we can now use `*` value to signify compatibility to _any_ stack.
3. Based on 2) this is still a Buildpack Author concern.

Reply:
I took the first sentence from the [PR description](https://github.com/buildpacks/spec/pull/155) but  I agree that your phrasing is much clearer. I'll go ahead and commit it.
Regarding 3), yes, you're totally right. I made a mistake at the beginning and forgot to fix it.

Reply:
Please see my changes in [this commit](https://github.com/buildpacks/docs/pull/279/commits/85fb1f1db86cb9637d7e6569af61afb42c91f00d).
I'll resolve this conversation. Please feel free to reopen it if you think that something is still missing.